### PR TITLE
Sourcing Result No Found

### PR DESCRIPTION
**Pull request by syriazapanta**

### CODE REVIEW

File: application/language/english/hirer/sourcing_lang.php [hunk 1]
```diff
@@ -34,5 +34,7 @@
 $lang['label_select_applicant_note'] = 'Select applicants to add to your job opening';
 $lang['searched_position'] = 'Because you\'ve searched for: <span><u class="text-danger">%s</u></span>';
 $lang['moved_successfully'] = 'The applicants you selected was moved to Sourced from Database folder under position %s.';
+$lang['note_no_search_found'] = 'We\'ve found nothing for the position: <strong><u class="text-info">%s</u></strong>';
```
Comment (princeryansy) [inline application/language/english/hirer/sourcing_lang.php]: ensure that html tags with attributes are not included in lang files.

File: application/modules/hirer/views/sourcing.php [hunk 2]
```diff
@@ -11,23 +11,35 @@
...
+                                        echo sprintf(
+                                            lang('searched_position'),
+                                            ucfirst(strtolower($positionTitle))
+                                        ); 
+                                        ?>
+                                    </p>
+                                </div>
+                            </div>
+                            <?php echo $manageMessage; ?>
+                            <?php echo form_open($formAction, 'id="form-job"'); ?>
```
Comment (princeryansy) [inline application/modules/hirer/views/sourcing.php]: ensure that the correct id name is used for the corresponding page.

**Comment by syriazapanta** [comment]
@princeryansy Please see updated changes. Thank you!

**Comment by princeryansy** [comment]
ACR Approved!

**Pull request merged**